api/envoy/extensions/filters/http/ratelimit/v3/rate_limit.proto

```diff
@@ -117,6 +361,27 @@ message RateLimitPerRoute {
     IGNORE = 2;
   }
 
+  enum RateLimitsOptions {
+    // Use the rate limits options defined in `VhRateLimitsOptions`.
+    DEFAULT = 0;
+
+    // Use the rate limits unless the route has a rate limit policy.
```

Comment:
This isn't just about the route; it's about whatever the next more specific override config is, whether that's route or cluster weight.

I suggest adding a comment above `RateLimitsOptions` explaining that this determines how the filter handles the case where there is an override config at multiple levels (virtual host, route, and cluster weight) for the same request.  Then the comment here can just say something like "Does not use this override if a more specific override exists." (and similar for the other enum values).

Reply:
Yea. I just copied the comment from old code.
Updated